src/core_modules/capture-core/components/BulkDataEntry/BulkDataEntryPlugin/BulkDataEntryPlugin.component.js

```diff
@@ -50,10 +51,11 @@ const BulkDataEntryPluginPlain = ({
                 configKey={configKey}
                 dataKey={dataKey}
                 onClose={onClose}
```

Comment:
We should change the interface a bit. A bit difficult since we don't have the plugin, but this is my initial suggestion:
- Remove `onClose`
-  Add `onComplete` and `onDefer`

When it comes to the app behaviour: 
- `onComplete` should remove the activeList and return to "origin"
- `onDefer` should just return to "origin"

Reply:
Great suggestion! I applied it to the https://github.com/dhis2/capture-app/pull/4000 PR as well